File: Dddml.Wms.JavaCommon/src/org/dddml/wms/domain/DocumentAction.java
package org.dddml.wms.domain;

import java.io.Serializable;

public class DocumentAction implements Serializable
{
    private String name;

    public String getName()
    {
        return this.name;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    DocumentAction ()
    {
    }

    public DocumentAction (String name)
    {
        this.name = name;
    }

    @Override
    public boolean equals(Object obj)
    {
        if (obj == this) {
            return true;
        }
        if (obj == null || obj.getClass() != this.getClass()) {
            return false;
        }

        DocumentAction other = (DocumentAction)obj;
        return true 
            && (name == other.name || (name != null && name.equals(other.name)))
            ;
    }

    @Override
    public int hashCode()
    {
        int hash = 0;
        if (this.name != null) {
            hash += 13 * this.name.hashCode();
        }
        return hash;
    }

}

File: Dddml.Wms.HttpServices.PhpClient/Dddml/Wms/Domain/DocumentAction.php
<?php

namespace Dddml\Wms\Domain;

use JMS\Serializer\Annotation\Type;
use Dddml\StringIdInterface;

class DocumentAction
{
    /**
     * @Type("string")
     */
    private $name;

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $name
     */
    public function setName($name)
    {
        $this->name = $name;
    }


}

